pkg/localcontroller/managers/lifelonglearning/lifelonglearningjob.go

```diff
@@ -671,50 +673,58 @@ func (lm *Manager) loadDataset(job *Job) error {
 		return fmt.Errorf("not exists dataset(name=%s)", datasetName)
 	}
 
-	jobConfig := job.JobConfig
-	jobConfig.DataSamples = &LLDataSamples{
-		Numbers:            0,
-		TrainSamples:       make([]string, 0),
-		EvalVersionSamples: make([][]string, 0),
-		EvalSamples:        make([]string, 0),
-	}
-
-	job.Dataset = dataset
+	job.JobConfig.Dataset = dataset
 	return nil
 }
 
 // initJob inits the job object
-func (lm *Manager) initJob(job *Job) error {
+func (lm *Manager) initJob(job *Job, name string) error {
+	var err error
+	job.JobConfig = new(JobConfig)
```

Comment:
This is not a good way to write code, it is recommended to:
```
jobConfig := new(JobConfig)
jobConfig.xxx = xxx
...
job.JobConfig = jobConfig
```

Reply:
ok, fix it